ajax/libs/Ventus/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Ventus",
+  "version": "0.2.1",
```

Comment:
The latest version is 0.2.2 according to https://github.com/rlamana/Ventus/releases/latest

I'll update the `package.json`.